dapps/assets/src/components/digitaltwins/DigitalTwins.vue

```diff
@@ -79,6 +79,7 @@
           no-local-sorting="true"
           @sort-changed="sortHandler"
           @scroll.native="scrollHandler"
+          @row-clicked="handleRowClicked"
```

Comment:
is the name for `@row-clicked` defined by the table component? I would prefer  `@row-click` analogue to vue event @click

Reply:
Yeah, it's in the boostrap-vue table